test/integration/targets/aws_ssm_parameters/tasks/main.yml

```diff
@@ -0,0 +1,102 @@
+---
+#
+#  Author: Michael De La Rue
+#  based on aws_lambda test cases
+- block:
+
+     # ============================================================
+     - name: Create or update key/value pair in aws parameter store
+       aws_ssm_parameter_store:
+         name: "/{{ssm_key_prefix}}/Hello"
+         description: "This is your first key"
+         value: "World"
+         ec2_access_key: '{{ec2_access_key}}'
```

Comment:
Not a blocker, but we've been using YAML anchors in tests to cut down on boilerplate.
```
  - name: set connection information for all tasks
    set_fact:
      aws_connection_info: &aws_connection_info
        aws_access_key: "{{ aws_access_key }}"
        aws_secret_key: "{{ aws_secret_key }}"
        security_token: "{{ security_token }}"
        region: "{{ aws_region }}"
    no_log: yes
```
and then you could replace credentials in tasks with:
```
aws_ssm_parameter_store:
  name: "/{{ssm_key_prefix}}/Hello"
  description: "This is your first key"
  value: "World"
  <<: *aws_connection_info
```